### PR TITLE
position fix university logo

### LINKED ISSUE #35: Correct position of University logo which is upside down
Correct position of **University logo** which is **upside down** in **Education section** of Header

Existing Element
```html
<img src="/masterPortfolio/static/media/iiitk_logo.c350d799.png" alt="IIITDM Kurnool" style="max-width: 100%; max-height: 100%; transform: scale(-0.5);">
```

Element replacing for **Correct position of Degrees Received Icon**
```html
<img src="/masterPortfolio/static/media/iiitk_logo.c350d799.png" alt="IIITDM Kurnool" style="max-width: 100%; max-height: 100%; transform: scale(0.5);">
```

You'll notice that _CSS style_ `transform` changed to to `scale(0.5)` which displays university logo (iiitk_logo.c350d799.png) in correct position instead of upside down.

### PR DESCRIPTION
**Pull request by mrjatinchauhan**
Finally, Resolves #35 Correct position of University logo which is upside down

### CODE REVIEW

**Comment by mrjatinchauhan** [comment]
Finally, Resolves #35 Correct position of University logo which is upside down

**Comment by ashutosh1919** [comment]
Please follow [this](https://stackoverflow.com/a/12093311/8289194) and synchronise your branch with remote branch to resolve merge conflicts

File: src/components/degreeCard/DegreeCard.js [hunk 1]
```diff
@@ -14,7 +14,11 @@ class DegreeCard extends Component {
               style={{
                 maxWidth: "100%",
                 maxHeight: "100%",
+<<<<<<< HEAD
+                transform: "scale(50%, 50%)",
```
Comment (ashutosh1919) [inline src/components/degreeCard/DegreeCard.js:18]: Please resolve this by removing the line which you don't want.

File: src/components/degreeCard/DegreeCard.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ class DegreeCard extends Component {
               style={{
                 maxWidth: "100%",
                 maxHeight: "100%",
+<<<<<<< HEAD
```
Comment (ashutosh1919) [inline src/components/degreeCard/DegreeCard.js:17]:
```suggestion
```

File: src/components/degreeCard/DegreeCard.js [hunk 3]
```diff
@@ -14,7 +14,11 @@ class DegreeCard extends Component {
...
+                transform: "scale(50%, 50%)",
+=======
```
Comment (ashutosh1919) [inline src/components/degreeCard/DegreeCard.js:19]:
```suggestion
```

File: src/components/degreeCard/DegreeCard.js [hunk 4]
```diff
@@ -14,7 +14,11 @@ class DegreeCard extends Component {
...
                 transform: "scale(-50%, -50%)",
```
Comment (ashutosh1919) [inline src/components/degreeCard/DegreeCard.js:20]:
```suggestion
```

File: src/components/degreeCard/DegreeCard.js [hunk 5]
```diff
@@ -14,7 +14,11 @@ class DegreeCard extends Component {
...
+>>>>>>> upstream/master
```
Comment (ashutosh1919) [inline src/components/degreeCard/DegreeCard.js:21]:
```suggestion
```

**Comment by ashutosh1919** [comment]
@mrjatinchauhan , any reason you closed this PR?

**Comment by mrjatinchauhan** [comment]
I will resolve this shortly definitely.

**Pull request closed without merging**